wiki/Tournaments/4WC/2024/en.md

```diff
@@ -0,0 +1,150 @@
+---
+tags:
+  - 4WC2024
+  - 4WC
+---
+
+# 4 Digit World Cup 2024
+
+![4WC banner](img/banner.jpg)
+
+The **4 Digit World Cup 2024** (***4WC2024***) is a 4v4, double-elimination, country-based osu! tournament hosted by ::{ flag=DE }:: [resoa](https://osu.ppy.sh/users/6754508) and ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899). It is the sixth instalment of the 4 Digit World Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-03-21/2024-04-13 (23:59 UTC) |
+| Team submission phase | 2024-04-14/2024-04-20 (23:59 UTC) |
+| Qualifier showcase | 2024-04-28 |
+| Qualifier stage | 2024-05-04/2024-05-05 |
+| Round of 32 | 2024-05-11/2024-05-12 |
+| Round of 16 | 2024-05-18/2024-05-19 |
+| Quarterfinals | 2024-05-25/2024-05-26 |
+| Semifinals | 2024-06-01/2024-06-02 |
+| Finals | 2024-06-08/2024-06-09 |
+| Grand Finals | 2024-06-15/2024-06-16 |
+
+## Prizes
+
+The 4 Digit World Cup 2024 awards winners with a [community-funded prize pool on Open Collective](https://opencollective.com/4wc).
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard for each player, Bluebunny tablet cover, 40% of the prize pool, profile badge (pending), animated profile banner |
```

Comment:
not sure if it sounds too clunky adding "for each player" on the tablet covers as well, but should probably not just have it on one of these prizes

Reply:
just forgot for some reason

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard and Bluebunny tablet cover for each player, 40% of the prize pool, profile badge (pending), animated profile banner |
```